src/components/map/Layers/Countries.tsx

```diff
@@ -0,0 +1,113 @@
+import { useState, useEffect, useContext } from "react"
+import { AppContext } from "context";
+import ReactDOMServer from "react-dom/server";
+import { EstimateGradePrevalence } from "types";
+import mapboxgl from "mapbox-gl";
+import CountryPopup from 'components/map/Popups/CountryPopup'
+
+function SetCountryEstimates(map: mapboxgl.Map, estimateGradePrevalences: EstimateGradePrevalence[]) {
+    estimateGradePrevalences.forEach((country: EstimateGradePrevalence) => {
+        if (country && country.testsAdministered && country.alpha3Code) {
+            map.setFeatureState(
+                {
+                    source: "Countries",
+                    sourceLayer: "Countries",
+                    id: country.alpha3Code,
+                },
+                {
+                    hasData: true,
+                    testsAdministered: country.testsAdministered,
+                    geographicalName: country.geographicalName,
+                    numberOfStudies: country.numberOfStudies,
+                    localEstimate: country.localEstimate,
+                    nationalEstimate: country.nationalEstimate,
+                    regionalEstimate: country.regionalEstimate,
+                    sublocalEstimate: country.sublocalEstimate,
+                }
+            );
+        }
+    });
+}
+
+function UpdateCountryEstimates(map: mapboxgl.Map, estimateGradePrevalences: EstimateGradePrevalence[]) {
+    var onlyIso3: (string | null)[] = estimateGradePrevalences.map((c) => { return c.alpha3Code })
```

Comment:
better to make all vars consts so that every variable is predictably block scoped rather than function scoped